src/lib/Microsoft.Health.Fhir.Ingest.Template/IContentTemplate.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.Health.Fhir.Ingest.Template
 {
-    public interface IContentTemplate
+    public interface IContentTemplate : ITemplate

```

Comment:
As with my comments in ITemplate, I think this is what needs to be a IValidatableObject.  NOt sure there is much merit extending that to the underlying templates.

Reply:
I updated it to have the wrapper wrapping the template and other contextual information.